internal/services/recoveryservices/recovery_services_vault_resource.go

```diff
@@ -140,6 +141,13 @@ func resourceRecoveryServicesVault() *pluginsdk.Resource {
 				Optional: true,
 				Default:  true,
 			},
+
+			"classic_vmware_replicate_enabled": {
```

Comment:
shouldn't; this be replication?
```suggestion
			"classic_vmware_replication_enabled": {
```
as that is what the sdk/api uses? as well makes the most sense

Reply:
oh make sense, just updated to rename, thanks!